### PR TITLE
Add html and css to my project

### PR DESCRIPTION
**Pull request by Rudy-Menekam**
The PR contains a simple page with header, frame and a form

### CODE REVIEW

**Comment by Rudy-Menekam** [comment]
As Feedback, I see you've tried following the professional guidelines on creating a project and pull request, except there are a few corrections to make
- Align the header properly
- Align the whole body to the center and fix the form
- Fix the stylelint error
- Give proper naming to the project and update the readme file too

**Comment by huseyinbiyik** [comment]
Hi @Rudy-Menekam,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there


## Required Changes ♻️
- Please use inline comment feature of Github when you are giving a review. Kindly check this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) to learn more about this. 
- Kindly make sure you are following the requirements below listed.
- - Check the errors
Does the reviewed project run on your machine?
Does the reviewed project work properly?
Are there any [linters errors](https://github.com/microverseinc/linters-config)?
Is the correct flow used?
Is the repository [documented in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)?
Are there any bugs that were missed by the author?
Is there any room for improvement in the project?
Check if you applied [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

- - Communicate kindly, descriptive and direct.
Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
Remember to explain why you request a change - it might be obvious for you but not for someone else.
Remember to be polite and clear in your comments - your job here is to help another person improve in a friendly way.
- Kindly make sure you added at least two comments in your review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rudy-Menekam** [comment]
hello @huseyinbiyik thanks for the review

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+
```
Comment (Rudy-Menekam) [inline index.html:13]: You should remove all unnecessary blank spaces in your code, so as to make the code look clean

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+#frame {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 700px;
+    margin-top: 50px;
+}
+
```
Comment (Rudy-Menekam) [inline css/styles.css:81]: the same rule applies here, you should remove all unnecessary spaces in your code

**Review by Rudy-Menekam — COMMENTED** [review]
You have done amazing work so far :+1: 
but you still have some minor changes to do 
**Required changes**

- Ensure your project works properly and has good alignments, it is not well aligned :EMOJICODE: 
- Fix your lint errors that are failing
- It will be good for you to document your repository in a professional way
- Just as the first point to better improve your work, you can center align your items, add some colors, horizontally align the header

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Rudy-Menekam 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✅ Quick attempt implementing requested change 👍
✅ Nice catching some bugs 👍 


 ## Required Changes ♻️
- It is very important to state to the student why you are requesting these changes, this will make the student see that you are not being biased with your comments
  
- Since you are pointing out a misalignment issue, I believe this has to do with the way a section renders on the browser if so, I think it would be nice to share a screenshot with the student for added specificity.

- I suggest you try to not sound too authoritative when requesting changes. Sounding politely will make it easier for the student to accept your suggestions.

## Optional suggestions
- Consider adding two to three inline comments as they are necessary when reviewing a project for clarity.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DuaneDave** [comment]
> You have done amazing work so far image but you still have some minor changes to do **Required changes**
> 
>     * Ensure your project works properly and has good alignments, it is not well aligned :EMOJICODE:
> 
>     * Fix your lint errors that are failing
> 
>     * It will be good for you to document your repository in a professional way
> 
>     * Just as the first point to better improve your work, you can center align your items, add some colors, horizontally align the header


- It is very important to state to the student why you are requesting these changes, this will make the student see that you are not being biased with your comments
  
- Since you are pointing out a misalignment issue, I believe this has to do with the way a section renders on the browser if so, I think it would be nice to share a screenshot with the student for added specificity.

- I suggest you try to not sound too authoritative when requesting changes. Sounding politely will make it easier for the student to accept your suggestions.

**Review by Rudy-Menekam — COMMENTED** [review]
You have done amazing work so far :+1: 
but you still have some minor changes to do 
**Required changes**

- Properly align your work, for example, can center align your items so it looks more presentable
- Try to fix the lint errors that are failing,
- It will be good for you to document your repository in a professional way :+1: 
- Just as the first point to better improve your work, you can center align your items(the form and frame(video)), add some colors, horizontally align the header(your menu items) 
the image below is what you currently have, so you can make the adjustments accordingly so as to make your work look more beautiful :+1: 
Screenshot from 2022-12-10 23-01-23

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Rudy-Menekam, 

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewers. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- Proper use of review features ✔️
- proper use of in-line comment ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**